`OpenSQL()` returns error when not using `nutsdb`

<!-- Thanks for filing an issue! Before submitting, please fill in the following information. -->

**What happened**
: 
When using `etcd` instead of `nutsdb`,
the Go code line `err := common.OpenSQL("../meta_db/dat/cbtumblebug.s3db")` causes error.


> ❯ make run 
> ./cb-tumblebug
> [CLOUD-BARISTA].[ERROR]: 2021-07-14 19:46:52 etcd-driver.go:214, github.com/cloud-barista/cb-store/store-drivers/etcd-driver.InitializeDriver() - context deadline exceeded 
> 
> 
> [Update system environment]
> 
> [Setup SQL Database]
> Database access info set successfully
> **unable to open database file: no such file or directory**
> panic: runtime error: invalid memory address or nil pointer dereference
> [signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x51fcbf]
> 
> goroutine 1 [running]:
> database/sql.(*Stmt).ExecContext(0x0, 0x166ad68, 0xc000130010, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
>         /snap/go/7954/src/database/sql/sql.go:2485 +0x5f
> database/sql.(*Stmt).Exec(...)
>         /snap/go/7954/src/database/sql/sql.go:2514
> github.com/cloud-barista/cb-tumblebug/src/core/common.CreateSpecTable(0x164ba60, 0xc000134008)
>         /home/jhseo/go/src/github.com/cloud-barista/cb-tumblebug/src/core/common/common.go:133 +0xa5
> main.main()
>         /home/jhseo/go/src/github.com/cloud-barista/cb-tumblebug/src/main.go:78 +0x612
> Makefile:12: recipe for target 'run' failed
> make: *** [run] Error 2


**What you expected to happen**
: 

**How to reproduce it (as minimally and precisely as possible)**
: 

**Anything else we need to know?**
: 

**Environment**
- Source version or branch: 
- OS: 
- Others: 

**Proposed solution**
: 

Add these lines before the `err := common.OpenSQL("../meta_db/dat/cbtumblebug.s3db")` line:

```Go
	err := os.MkdirAll("../meta_db/dat/", os.ModePerm)
	if err != nil {
		fmt.Println(err.Error())
	}
```

**Any other context**
:

--- a/src/main.go
+++ b/src/main.go
@@ -59,7 +59,13 @@ func main() {
 	//Setup database (meta_db/dat/cbtumblebug.s3db)
 	fmt.Println("")
 	fmt.Println("[Setup SQL Database]")
-	err := common.OpenSQL("../meta_db/dat/cbtumblebug.s3db")
+
+	err := os.MkdirAll("../meta_db/dat/", os.ModePerm)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+
+	err = common.OpenSQL("../meta_db/dat/cbtumblebug.s3db")
 	if err != nil {
 		fmt.Println(err.Error())
 	} else {

